perllib/FixMyStreet/Cobrand/Bexley.pm

```diff
@@ -44,6 +44,12 @@ sub geocode_postcode {
 
 sub disable_resend_button { 1 }
 
+# Bexley uses a separate waste/submit.(html|txt) template for waste reports
```

Comment:
I'm not happy with my method names, and am not sure if I've overengineered this (rather than just sticking 'if bexley && waste' checks in Email.pm, but that doesn't seem good either).

Reply:
I wouldn't change the code at all in this commit, I think; I would have an IF at the top of the fixmystreet.com/submit.txt and html file specially for Bexley - which matches the `other-reported` way of doing it, and we don't know at this point if we'll need anything else.

Reply:
OHHHH
There's a bunch of if-else PROCESS statements 🤦‍♀️